viewer/package.json

```diff
@@ -23,13 +23,15 @@
     "isomorphic-fetch": "^2.2.0",
     "jquery": "^2.2.0",
     "leaflet": "^0.7.7",
+    "leaflet-draw": "^0.2.3",
     "less": "^2.5.3",
     "less-loader": "^2.2.2",
     "lodash": "^4.0.0",
     "react": "^0.14.3",
     "react-bootstrap": "^0.28.2",
     "react-dom": "^0.14.3",
-    "react-leaflet": "^0.10.0",
+    "react-leaflet": "^0.9.0",
+    "react-leaflet-draw": "^0.5.0",
     "react-redux": "^4.0.0",
```

Comment:
Getting:

```console
ERROR in ./utils/renderRoads.js
Module not found: Error: Cannot resolve module 'geocolor' in /Users/eugene/proj/geotrellis-osm-elevation/viewer/utils
 @ ./utils/renderRoads.js 8:15-34
^C
```
I had to install the module manually
```
npm install --save geocolor
```
This change should be reflected here as well.